devices/src/tpm.rs

```diff
@@ -28,57 +27,132 @@ pub enum Error {
 }
 type Result<T> = anyhow::Result<T, Error>;
 
+#[allow(dead_code)]
+enum LocStateFields {
+    TpmEstablished,
+    LocAssigned,
+    ActiveLocality,
+    Reserved,
+    TpmRegValidSts,
+}
+
+enum LocStsFields {
```

Comment:
don't you need the `dead_code` lint for rest of the enums here?

Reply:
Yes